flow360/component/simulation/outputs/output_entities.py

```diff
@@ -41,6 +42,7 @@ class Slice(_SliceEntityBase):
     """Slice output item."""
 
     private_attribute_entity_type_name: Literal["Slice"] = pd.Field("Slice", frozen=True)
+    private_attribute_id: str = pd.Field(default=str(uuid.uuid4()), frozen=True)
```

Comment:
I think every instance will have exactly the same uuid. Default factory should be used instead